diff --git a/MarketScanner.UI.Wpf2/Views/MainWindow.xaml.cs b/MarketScanner.UI.Wpf2/Views/MainWindow.xaml.cs
index 039dfa26d86763a01d568efaae62a599dd51cf92..9b6b2c27988d60aa1fead60069755e70fb699faf 100644
--- a/MarketScanner.UI.Wpf2/Views/MainWindow.xaml.cs
+++ b/MarketScanner.UI.Wpf2/Views/MainWindow.xaml.cs
@@ -1,20 +1,12 @@
-﻿using System.Threading.Tasks;
 using System.Windows;
-using System.Windows.Media.Animation;
-using System.Windows.Controls;
-using MarketScanner.UI.Wpf.ViewModels;
-using System.Diagnostics;
 
 namespace MarketScanner.UI.Views
 {
-public partial class MainWindow : Window
-{
-    public MainWindow()
+    public partial class MainWindow : Window
     {
-        InitializeComponent();
-        DataContext = new MainViewModel();
-
-        PresentationTraceSources.DataBindingSource.Switch.Level = SourceLevels.Critical;    }
-}
-
+        public MainWindow()
+        {
+            InitializeComponent();
+        }
+    }
 }
